config/initializers/devise.rb

```diff
@@ -13,4 +13,6 @@
 
   # rememberable
   config.remember_for = 12.weeks
+
+  config.secret_key = ENV["SECRET_KEY"]
```

Comment:
To make this easier to use in dev, it might make sense to do something like:

```
config.secret_key = if Rails.env.production?
  ENV.fetch("SECRET_KEY")
else
  ENV.fetch("SECRET_KEY", "secretkey")
end
```

I'm not sure if this is a terrible idea or not.